src/features/post/components/Post.tsx

```diff
@@ -81,6 +85,54 @@ export const Post: FC<PostProps> = ({post}: PostProps) => {
             <div className="mx-10">
               <Reader>{post?.content!}</Reader>
             </div>
+            <div className="flex items-stretch justify-self-center">
+              <div
+                className="mx-5 flex items-stretch justify-self-center"
+                onClick={() =>
+                  reactToPost(post.id as string, ReactionType.LIKE)
+                }
+              >
+                <svg
+                  className="h-6 w-6 hover:bg-gray-400 focus:bg-gray-400 focus:outline-none focus:ring active:bg-gray-400 dark:text-white"
+                  aria-hidden="true"
+                  xmlns="http://www.w3.org/2000/svg"
+                  fill="currentColor"
+                  viewBox="0 0 24 24"
+                >
+                  <path
+                    fill-rule="evenodd"
+                    d="M9 14.3H5a2 2 0 0 1-1.6-.9 2 2 0 0 1-.3-1.8l2.4-7.2C5.8 3.5 6 3 7.4 3c2 0 4.2.7 6.1 1.3l1.4.4v9.8a32 32 0 0 0-4.2 5.5c-.1.4-.5.7-.9.9a1.7 1.7 0 0 1-2.1-.7c-.2-.4-.3-.8-.3-1.3L9 14.3Zm10.8-.3H17V6a2 2 0 1 1 4 0v6.8c0 .7-.5 1.2-1.2 1.2Z"
+                    clip-rule="evenodd"
+                  />
+                </svg>
+                <label data-testid="like-reaction" className="ml-2">
+                  {post.reactions?.likes}
+                </label>
+              </div>
+              <div
```

Comment:
divs is not an interactive element
instead use button (check shadcn-ui icon btn)